components/canvas/Node.module.scss

```diff
@@ -85,17 +94,20 @@
   padding: 10px;
 }
 
-.description {
+.node__description {
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 5;
```

Comment:
Is this needed if you extend `.node__info-text`?

Reply:
Yes, webkit is a solution to apply text-overflow on vertical overflows.